pkcs7/src/lib.rs

```diff
@@ -15,6 +15,9 @@
     unused_qualifications
 )]
 
+#[cfg(feature = "std")]
+extern crate std;
```

Comment:
This seems irrelevant now?

Reply:
Yes. I thought I had reverted all changes in pkcs7. Don't know why this was still in.